src/v2/Components/Artwork/Details.tsx

```diff
@@ -131,17 +131,11 @@ export class Details extends React.Component<Props, null> {
     }
   }
 
-  saleLine() {
-    const artwork = this.props.artwork
-    const hasSaleMessage =
-      artwork.sale_message && artwork.sale_message !== "Contact For Price"
-    const notInAuction = !(artwork.sale && artwork.sale.is_auction)
-    if (hasSaleMessage && notInAuction) {
-      return <div>{artwork.sale_message}</div>
-    }
-  }
-
   saleInfoLine() {
+    const { showSaleLine } = this.props
```

Comment:
Interestingly, I noticed that `showSaleLine` is defined as a prop but never used, so I wired that up. That may affect other artwork rails.

Reply:
Yeah, actually this definitely does and needs to be addressed before merging.

Reply:
We should set a defaultProp on the class with `showSaleLine = true` so that we can ensure backward compatibility , or explicitly set it where necessary where `<Details />` is being used around app. Default prop would probably be safest tho